Sources/YCoreUI/Extensions/UIKit/UIView+constrainCenter.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  UIView+constrainCenter.swift
+//  YCoreUI
+//
+//  Created by Dharmik Ghelani on 20/10/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    /// Struct to define center attribute
+    struct Center: OptionSet {
+        let rawValue: UInt
+        static let x = Center(rawValue: 1 << 0)
+        static let y = Center(rawValue: 1 << 1)
+        static let all: Center = [.x, .y]
+    }
+    
+    /// Constrain the receiving view with provided center attributes
+    /// - Parameters:
+    ///   - center: Center attribute of view to constrain to (default `.all`)
+    ///   - view2: View object to which constrain to (pass `nil` to constrain to superview)
+    ///   - relation: Relation to evaluate (default `.equal`)
+    ///   - offset: Offset to apply to attribute(center X and Y) (default `.zero`)
+    ///   - priority: Constraint priority (default `.required`)
+    ///   - isActive: Whether to activate the constraint or not (default `true`)
+    /// - Returns: The created layout constraint
```

Comment:
```suggestion
    /// Constrain the center of the receiving view with the center of another view
    /// - Parameters:
    ///   - center: which center attributes to constrain (default `.all`)
    ///   - view2: view or layout guide to constrain to (pass `nil` to constrain to superview)
    ///   - relation: relation to evaluate (towards view2) (default `.equal`)
    ///   - offset: offset to apply relative to view2.center (default `.zero`)
    ///   - priority: constraint priority (default `.required`)
    ///   - isActive: whether to activate the constraints or not (default `true`)
    /// - Returns: dictionary of constraints created, keyed by `.centerX, .centerY`
```

Stylistically we've begin parameter descriptions with lowercase letters. And where it makes sense we should use identical language to existing constrain method parameters.

Reply:
👍🏻Addressed the change